app/src/main/java/com/example/cinema/RetrofitService.kt

```diff
@@ -45,5 +56,18 @@ object RetrofitService {
             level = HttpLoggingInterceptor.Level.BODY
         }
     }
+    interface MovieApi {
+        @GET("movie/popular")
+        fun getPopularMovies(
+            @Query("api_key") apiKey: String = "753b84576c954d96997803298a188f83",
+            @Query("page") page: Int
+        ): Call<MovieResponse>
+
+        @GET("movie/{movie_id}")
+        fun getMovieById(@Path("movie_id") movieId: Int=1,
+                         @Query("api_key") apiKey: String = "753b84576c954d96997803298a188f83")
+                :Call<MoviesData>
+    }
+
 }
```

Comment:
we already have similar interface, shouldn't you just put all http methods to one interface?